exercises/practice/complex-numbers/.meta/test_template.ecr

```diff
@@ -0,0 +1,77 @@
+require "spec"
+require "../src/*"
+
+<% property_map = {
+  ".real" => ".real",
+  ".imaginary" => ".imaginary",
+  ".abs" => ".abs",
+  ".conjugate" => ".conjugate",
+  ".exp" => ".exp",
+  ".add" => "+",
+  ".sub" => "-",
+  ".mul" => "*",
+  ".div" => "/"
+} %>
+
+<%
+complex_args = ->(json : JSON::Any) {
+  json
+    .as_a
+    .map(&.to_s)
+    .map(&.gsub("e", "Math::E"))
+    .map(&.gsub("pi", "Math::PI"))
+    .map(&.gsub("ln", "Math.log"))
+    .join(",")
+}
```

Comment:
I had to use a proc literal for this because I was getting errors trying to use `def`. I'm not sure if this is expected or if I was doing something wrong.